app/subscriber/src/features/search-page/components/advanced-search/components/sections/DateSection.tsx

```diff
@@ -16,28 +16,10 @@ export const DateSection: React.FC = () => {
     { storeSearchFilter: storeFilter },
   ] = useContent();
 
-  // disable quick picker when user selects a date on react-date-picker
-  const [disableQuickPick, setDisableQuickPick] = React.useState(
```

Comment:
Why do we need to disable the offset picker?  I removed all this logic to get rid of bugs